modules/ROOT/pages/setup.adoc

```diff
@@ -25,10 +25,14 @@ Roles are allocated via the Access Management section in Anypoint Platform.
 
 === Partner Manager related technical requirements:
 
-* Mule runtime 4.2.2 or higher. Make sure you have the required Mule runtime capacity to handle your B2B transaction workload needs.
-If you are unsure about the Mule runtime capacity required to handle your B2B transactions, contact your MuleSoft account representative for assistance.
-
-* A storage solution to store the B2B transmission payloads you send or receive. Storage options can include a Database, Amazon S3, Azure Blob Storage, and so on.
+* Mule runtime 4.2.2 or higher.
```

Comment:
GLOBALLY REPLACE any other variation of "Mule runtime engine" or "Mule" with one of those (only two) accepted presentations. Doing so should be muscle-memory by now :-)

Reply:
```suggestion
* Mule runtime engine (Mule) 4.2.2 or later
```
Not "higher": "later." See style ref.
No period.

Reply:
Fixed.